### PR TITLE
Mm 50013 - readd invite members screen cloud version

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR modifies the invite members section so in the cloud version it will allow the user to invite members directly by sending the email invitations, also shows the copy invitation link and the skip button (which should load after 3 seconds)

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50013

#### Screenshots

https://user-images.githubusercontent.com/10082627/232589102-5a1543d5-c209-4e17-babf-e5def1b7696b.mov

Screenshot 2023-04-17 at 21 20 35

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by neallred — APPROVED** [review]
LGTM, thanks @pvev !

File: webapp/channels/src/components/preparing_workspace/invite_members.test.tsx [hunk 1]
```diff
@@ -68,4 +84,16 @@ describe('InviteMembers component', () => {
         fireEvent.click(button);
         expect(defaultProps.next).toHaveBeenCalled();
     });
+
+    it('shows send invites button when in cloud', () => {
```
Comment (marianunez) [inline webapp/channels/src/components/preparing_workspace/invite_members.test.tsx:88]: Should we add the opposite test to not show if in Self-hosted?

File: webapp/channels/src/components/preparing_workspace/preparing_workspace.tsx [hunk 2]
```diff
@@ -225,16 +227,15 @@ const PreparingWorkspace = (props: Props) => {
         const sendFormStart = Date.now();
         setSubmissionState(SubmissionStates.Submitting);
 
-        if (form.organization && !isSelfHosted) {
+        if (!form.teamMembers.skipped && !isConfigSiteUrlDefault && !isSelfHosted) {
```
Comment (marianunez) [inline webapp/channels/src/components/preparing_workspace/preparing_workspace.tsx:230]: I guess it doesn't hurt this additional check but in cloud, the SIteURL is always configured to their workspace url at this point in the onboarding.

File: webapp/channels/src/components/preparing_workspace/plugins.tsx [hunk 3]
```diff
@@ -46,32 +45,19 @@ const Plugins = (props: Props) => {
         className += ' ' + props.className;
     }
 
-    let title = (
+    const title = (
         <FormattedMessage
-            id={'onboarding_wizard.cloud_plugins.title'}
-            defaultMessage='Welcome to Mattermost!'
+            id={'onboarding_wizard.self_hosted_plugins.title'}
```
Comment (marianunez) [inline webapp/channels/src/components/preparing_workspace/plugins.tsx:50]: nit: we should rename this text id name to remove the self-hosted reference if it's now for both cases

```suggestion
            id={'onboarding_wizard.self_hosted_plugins.title'}
```

**Review by marianunez — APPROVED** [review]
Thanks @pvev! Some non-blocking comments below

**Comment by esethna** [comment]
@pvev looking great! How are invalid emails handled in this screen? 

@annelieseklein do you know how we handled such error states with this invite screen before it was turned off? We probably don't want to block them getting into the product, but we also want them to know the invite was not sent

**Comment by annelieseklein** [comment]
I don't see any examples of error states but i do see invitations sent  - i can probably make a similar state for invalid issues. 

Screen Shot 2023-04-17 at 5 40 02 PM

Here is an example we could do for a general error message, cc: @esethna 
Screen Shot 2023-04-17 at 5 48 04 PM

**Review by JulienTant — APPROVED** [review]
Looks good @pvev thank you!

**Comment by pvev** [comment]
I was able to spin up a server using a dummy pr and was able to validate the e2e flow:


https://user-images.githubusercontent.com/10082627/233186295-fe137ece-f69c-421f-93dc-971ddced9ca1.mov

Screenshot 2023-04-19 at 21 59 29

**Comment by stevemudie** [comment]
@pvev Unfortunately, both the formatting and the link are still like before? 🤔 

**Note**: I mistakenly clicked the 'No thanks' link which dismissed the initial task list. 🤦🏻‍♂️ 
So I bumped up a member I'd invited to admin then when I went through the steps there (in incognito) it was where it all looks and acted as before.

**Comment by stevemudie** [comment]
@pvev There was a failed test in [the E2E results](https://automation-dashboard.vercel.app/cycle/328019) that you may want to take a look at. It involves adding members to a channel  which isn't what this PR is about of course but do you think it was effected by changes made here?

**Comment by pvev** [comment]
> @pvev There was a failed test in [the E2E results](https://automation-dashboard.vercel.app/cycle/328019) that you may want to take a look at. It involves adding members to a channel which isn't what this PR is about of course (it will make more sense when you see the failed test) but do you think it was effected by changes made here?

hey @stevemudie not at all, this initial flow modification has nothing to do with the failed test, maybe was a flaky tests that unfortunately failed in this run too?

**Comment by pvev** [comment]
@stevemudie there is a working spinwick server here to finish the validation for this PR.

**Comment by pvev** [comment]
> https://mattermost.atlassian.net/browse/MM-50013

ticket created for this one: https://mattermost.atlassian.net/browse/MM-52454

**Pull request merged**